perllib/FixMyStreet/Cobrand/Bexley/Garden.pm

```diff
@@ -30,14 +35,27 @@ sub garden_current_subscription {
     my $uprn = $self->{c}->stash->{property}{uprn};
     return undef unless $uprn;
 
-    my $is_free = $self->agile->IsAddressFree($uprn);
-    return undef if $is_free->{IsFree} eq 'True';
+    my $results = $self->agile->CustomerSearch($uprn);
```

Comment:
I think we mentioned doing a check against the current subscription that is in the DB and making sure the customer email & external_id match the Customer and ServiceContract returned from Agile.
I'm not sure if it would be better to do this match as part of your PR, as strictly speaking it's not necessary to get cancellation working (am also worried about me making too many changes on my PR that then mess with your work here). I think get_original_sub() in Controller/Waste.pm needs updating so it searches on Bexley's `uprn` field as well as `property_id`.